pyproject.toml

```diff
@@ -889,3 +889,12 @@ ignore_imports = [
     # TODO: remove once npe1 deprecated
     "napari._qt.qt_main_window -> napari._qt._qplugins",
 ]
+
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git*,*.svg,*.rtf,*.rtf.tmpl,.venv,./tools/string_list.json'
+check-hidden = true
+# lines with é -- non-English
+ignore-regex = '(.*[àéç].*|^\s*"image/\S+": ".*|\biTerm\b)'
```

Comment:
Question on the regex: Why are we excluding image keys? Could iTerm be moved into the ignore-words-list?